temporal-sdk/src/main/java/io/temporal/internal/worker/ActivityPollTask.java

```diff
@@ -75,14 +75,6 @@ public PollActivityTaskQueueResponse poll() {
               .build());
     }
 
-    if (taskQueueActivitiesPerSecond > 0) {
-      pollRequest.setTaskQueueMetadata(
-          TaskQueueMetadata.newBuilder()
-              .setMaxTasksPerSecond(
-                  DoubleValue.newBuilder().setValue(taskQueueActivitiesPerSecond).build())
-              .build());
-    }
-
```

Comment:
Why is this removed? Task queue level rate limiting is a very important feature

Reply:
There were two identical blocks, I've assumed that it was a mistake and removed one.